script.js

```diff
@@ -1,4 +1,398 @@
+      return `Computer shouted: muk-jji-ppa!`;
+    }
+    if (mukJjiPpaState == "winner wins") {
+      var outputOne = `It is a draw!<br><br>Program Win Count: ${programWinCount}.<br>Program Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      if (tempWinner == "player") {
+        playerWinCount = playerWinCount + 1;
+        return `You win!<br><br>Player Win Count: ${playerWinCount}.<br>Player Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      }
+      if (tempWinner == "computer") {
+        computerWinCount = computerWinCount + 1;
+        return `You lose!<br><br>Player Win Count: ${playerWinCount}.<br>Player Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      }
+      return outputOne;
+    }
+  }
+};
+
+var mukJjiPpaWinLoseBoardForComputerVSComputer = function (programToComputer) {
+  mukJjiPpaState = "play again";
+  if (mukJjiPpaState == "play again") {
+    if (
+      programToComputer == ["scissors" + "scissors"] ||
+      programToComputer == ["paper" + "paper"] ||
+      programToComputer == ["stone" + "stone"] ||
+      programToComputer == ["reversed scissors" + "scissors"] ||
+      programToComputer == ["reversed paper" + "paper"] ||
+      programToComputer == ["reversed stone" + "stone"]
+    ) {
+      mukJjiPpaState = "winner wins";
+      totalCount = totalCount + 1;
+    }
+
+    if (
+      programToComputer == ["scissors" + "paper"] ||
+      programToComputer == ["paper" + "stone"] ||
+      programToComputer == ["stone" + "scissors"] ||
+      programToComputer == ["reversed stone" + "paper"] ||
+      programToComputer == ["reversed scissors" + "stone"] ||
+      programToComputer == ["reversed paper" + "scissors"]
+    ) {
+      tempWinner = "program";
+      return `Program shouted: muk-jji-ppa!`;
+    }
+
+    if (
+      programToComputer == ["stone" + "paper"] ||
+      programToComputer == ["scissors" + "stone"] ||
+      programToComputer == ["paper" + "scissors"] ||
+      programToComputer == ["reversed scissors" + "paper"] ||
+      programToComputer == ["reversed paper" + "stone"] ||
+      programToComputer == ["reversed stone" + "scissors"]
+    ) {
+      tempWinner = "computer";
+      return `Computer shouted: muk-jji-ppa!`;
+    }
+    if (mukJjiPpaState == "winner wins") {
+      var outputOne = `It is a draw!<br><br>Program Win Count: ${programWinCount}.<br>Program Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      if (tempWinner == "program") {
+        programWinCount = programWinCount + 1;
+        return `The program won!<br><br>Program Win Count: ${programWinCount}.<br>Program Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      }
+      if (tempWinner == "computer") {
+        computerWinCount = computerWinCount + 1;
+        return `The program lost!<br><br>Program Win Count: ${programWinCount}.<br>Program Lose Count: ${computerWinCount}.<br>Draw Count: ${draw}`;
+      }
+      return outputOne;
+    }
+  }
+};
+
```

Comment:
since your code is quite long, it would be nice to see a few more comments in the code which I think would make it easier to read. However, do keep up with the good variable naming habits!